src/frontend/src/pages/shared-components/appbar.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import {
+    Toolbar,
+    Grid,
+    IconButton,
+    Typography,
+    Button,
+    AppBar,
+    Box
+} from "@material-ui/core";
+import AccountCircle from "@material-ui/icons/AccountCircle";
+import { Link } from "react-router-dom";
+
+export const ApplicationBar: React.FC = () => {
+    return (
+        <AppBar position="static">
+            <Toolbar>
+                <Grid justify="space-between" alignItems="center" container>
+                    <Grid item>
+                        <Grid
+                            justify="space-evenly"
+                            alignItems="center"
+                            container
+                        >
+                            <Grid item>
+                                <IconButton edge="start" color="inherit">
+                                    <AccountCircle />
+                                </IconButton>
+                            </Grid>
+                            <Grid item>
+                                <Typography display="inline">
+                                    Logged in As Tina
+                                </Typography>
+                            </Grid>
+                            <Grid item>
+                                <Box p={3}>
+                                    <Typography variant="h6" display="inline">
+                                        Points: 0
+                                    </Typography>
+                                </Box>
+                            </Grid>
+                        </Grid>
+                    </Grid>
+                    <Grid item>
+                        <Link to="/login" style={{ textDecoration: "none" }}>
+                            <Button variant="contained" size="large">
```

Comment:
The button shouldn't be size large. That's why the toolbar is so tall. 
```suggestion
                            <Button variant="contained">
```